### PR TITLE
Update minio and mc to latest release

### PR DESCRIPTION
**Pull request by nitisht**
**What this PR does / why we need it**: Update minio and mc Docker images to latest release

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/minio/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Minio is a high performance distributed object storage server, designed for large-scale private cloud infrastructure.
 name: minio
-version: 1.7.0
```
Comment (cpanato) [inline stable/minio/Chart.yaml:4]: @nitisht new release, please bump to `1.8.0`

Comment (nitisht) [reply stable/minio/Chart.yaml:4]: @cpanato Minio makes release pretty frequently, sometimes multiple releases per week. So, we generally make minor version updates in the chart - unless this chart itself is getting any significant feature update. I think `1.7.1` is fine.

Comment (cpanato) [reply stable/minio/Chart.yaml:4]: I bumped in another PR that fixes the stateful sets, so you can close this one here

**Pull request closed without merging**